src/pages/guidelines/themes/code.mdx

```diff
@@ -49,10 +48,18 @@ yarn add @carbon/themes
 
 ## Usage
 
-You can use `@carbon/themes` in JavaScript or Sass by including this package in
-your project. By default, `@carbon/themes` provides a set of color tokens that
-are pre-defined for a specific theme. Currently, we offer the following color
-themes: white, gray 10, gray 90, gray 100 .
+There are three ways to utilize Carbon Themes. First, and maybe the simplest way
+is to bring in our
+[`<Theme>`](https://react.carbondesignsystem.com/?path=/story/components-theme--default)
+component and pass your desired theme as a prop. Second, if you need more
+granular control of what gets themed and when and you're using `@carbon/styles`
+or `@carbon/react` you already have access to sass theming with no need for an
+extra package. Thirdly, can use `@carbon/themes` in JavaScript or Sass by
+including this package in your project.
+
+By default, `@carbon/themes` provides a set of color tokens that are pre-defined
+for a specific theme. Currently, we offer the following color themes: white,
+gray 10, gray 90, gray 100 .
 
 You can preview all of the token values for this on the
 [Carbon Design System website](https://www.carbondesignsystem.com/guidelines/color/usage)
```

Comment:
Extra change ... they're already on the website for this is so you don't need to say the website. 
```suggestion
[color guidelines(https://www.carbondesignsystem.com/guidelines/color/usage) page.
```